.golangci.yml

```diff
@@ -167,21 +166,21 @@ linters:
     - prealloc
     - scopelint
     - testpackage
+    - varnamelen
     - wrapcheck
   disable-all: false
   # presets:
   #   - bugs
   #   - unused
   fast: false
 
-
 issues:
   # List of regexps of issue texts to exclude, empty list by default.
   # But independently from this option we use default exclude patterns,
   # it can be disabled by `exclude-use-default: false`. To list all
   # excluded by default patterns execute `golangci-lint run --help`
   exclude:
-    - 'can be .*testify/assert.TestingT'
+    - "can be .*testify/assert.TestingT"
```

Comment:
Auto formatting? 😄 Either way would be nice if all the rows in this list (and quotes in this file) were consistent.

Reply:
Sure! I'll fix that as well!

Reply:
Well it wasn't possible to have `"` on the quotes below because the content would then need to be escaped. Having `"` unless otherwise needed seems to be some kind of standard.